plugins/modules/mysql_user.py

```diff
@@ -746,8 +746,6 @@ def pick(x):
         privileges = [pick(x.strip()) for x in privileges]
         if "WITH GRANT OPTION" in res.group(7):
             privileges.append('GRANT')
```

Comment:
```suggestion
            privileges.append('GRANT')
        if 'REQUIRESSL' in res.group(7) and not tls_requires:
            privileges.append('REQUIRESSL')
```

Reply:
Mind that `tls_requires` is not defined in this context

Reply:
Yeah, I think that makes sense.  I'll try to come up with on updated patch.

Reply:
When you update this, can you also add tests to verify the behaviour, both in tasks with `priv` and with `tls_requires`?